src/lifecycle.mjs

```diff
@@ -285,6 +289,10 @@ export async function init(worker, crawlPath) {
     for await (const strategy of segment) {
       if (strategy.extractor) {
         const extractStrategy = finder("extraction", strategy.name);
+        const outputPath = generatePath(
```

Comment:
OK so if I understand correctly, then defining "outputFilename" in the crawlPath is optional, right? And if it isn't defined then we fall back on the regular schema of strategyname+type That's good!

Reply:
You are right.